eng/common/scripts/Helpers/Resource-Helpers.ps1

```diff
@@ -19,18 +19,33 @@ function Get-PurgeableGroupResources {
 }
 
 function Get-PurgeableResources {
+    $purgeableResources = @()
     $subscriptionId = (Get-AzContext).Subscription.Id
 
+    Write-Verbose "Retrieving the purgeable resources from subscription $subscriptionId"
+
+    Write-Verbose "Retrieving deleted Key Vaults from subscription $subscriptionId"
+
     # Get deleted Key Vaults for the current subscription.
-    Get-AzKeyVault -InRemovedState `
+    $deletedKeyVaults  = Get-AzKeyVault -InRemovedState `
```

Comment:
Given the error in the pipeline, add `-ErrorAction Ignore` to both this `Get-AzKeyVault` call and the one above. If the RG was deleted between getting the RG to delete and now, this errs. Using `Ignore` will just ignore the error and not emit any value.